SlingModelPersist/src/main/java/org/apache/sling/models/persist/ModelPersist.java

```diff
@@ -1,21 +1,21 @@
-package org.apache.sling.models.persist;
-
-import javax.jcr.RepositoryException;
-import org.apache.sling.api.resource.PersistenceException;
-import org.apache.sling.api.resource.ResourceResolver;
-import org.jetbrains.annotations.NotNull;
-
-/**
- * Definition of JCR Persist service
- */
-public interface JcrPersist {
-
-    void persist(@NotNull Object object, @NotNull ResourceResolver resourceResolver) throws RepositoryException, PersistenceException, IllegalArgumentException, IllegalAccessException;
-
-    void persist(@NotNull Object object, @NotNull ResourceResolver resourceResolver, boolean deepPersist) throws RepositoryException, PersistenceException, IllegalArgumentException, IllegalAccessException;
-
-    void persist(String nodePath, @NotNull Object object, @NotNull ResourceResolver resourceResolver) throws RepositoryException, PersistenceException, IllegalArgumentException, IllegalAccessException;
-
-    void persist(String nodePath, @NotNull Object object, @NotNull ResourceResolver resourceResolver, boolean deepPersist) throws RepositoryException, PersistenceException, IllegalArgumentException, IllegalAccessException;
-    
-}
+package org.apache.sling.models.persist;
+
+import javax.jcr.RepositoryException;
+import org.apache.sling.api.resource.PersistenceException;
+import org.apache.sling.api.resource.ResourceResolver;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Definition of JCR Persist service
+ */
+public interface ModelPersist {
```

Comment:
For some reason the name ModelPersist seems... off? PersistModel, ModelPersistence?

Reply:
Well I think you might be right.  Class names work better as nouns, and this is more verb-y.  ModelPersistor is something which persists models, maybe that's a better name?